wordpress/wp-content/plugins/sil-dictionary-webonary/webonary/Webonary_Cloud.php

```diff
@@ -38,7 +44,15 @@ private static function remoteGet($path, $params = array()) {
 			error_log('Getting results from ' . $url);
 		}
 
-		return wp_remote_get($url);
+		$response = wp_remote_get($url);
+
+		if (is_wp_error($response)) {
+			error_log($response->get_error_message());
+			return;
```

Comment:
There should be no empty return statements in this function. They should be `return null;` or `return '';`.

Reply:
done :-) thanks.